pom.xml

```diff
@@ -4,8 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.86</version>
-    <relativePath/>
```

Comment:
Why deviate from the archetype in this regard?

https://github.com/jenkinsci/archetypes

Reply:
Hmm, I don’t remember why now. I may have just missed that this was part of the archetype. I’ll add it back tomorrow (no in front of my dev computer right now).